src/Shim.php

```diff
@@ -751,4 +751,28 @@ public static function unixToJd($timestamp) {
 			return self::GregorianToJd(gmdate('n', $timestamp), gmdate('j', $timestamp), gmdate('Y', $timestamp));
 		}
 	}
+	
+	/**
+	 * By default, the result of A % B will be of the sign of A.
+	 * This is not necessarily the agreed version of the mod operator by other languages.
+	 * This mod function returns always the positive result of the modulo operation.
+	 * 
+	 * For example: 
+	 *    (Standard PHP)   -3 % 4 = -3
+	 *    (Shim)           Shim::mod(-3, 4) = 1
+	 * 
+	 * @param number $dividend
+	 * @param number $divisor
+	 * @return number
+	 */
+	public static function mod($dividend, $divisor) {
+	    if ($dividend == 0 || $divisor == 0) return 0;
```

Comment:
`$dividend == 0` is not a problem.  For efficiency, remove this check?

Can we use `===` instead of `==`?